engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -856,58 +853,6 @@ void _testLongestIncreasingSubsequence() {
   });
 }
 
-void _testText() {
-  test('renders a piece of plain text', () async {
```

Comment:
should we still keep this test case?

Reply:
Text is covered in a dedicated test file:

https://github.com/flutter/flutter/blob/master/engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_text_test.dart

This code here doesn't add value.